shared/src/test/kotlin/com/egm/stellio/shared/model/CompactedEntityTests.kt

```diff
@@ -1,12 +1,147 @@
 package com.egm.stellio.shared.model
```

Comment:
in this test class, review the indentation of all the JSON payloads, they are very hard to read